RELEASE.txt

```diff
@@ -59,10 +59,14 @@ FIXES
   arguments would yield an exception. This issue was found via qt4 and qt5 tools in
   scons-contrib https://github.com/SCons/scons-contrib/issues/45
 - Fix issue where if you only had mingw installed on a Windows system and no MSVC compiler, and
-  did not explicitly request the mingw tool, mingw tool initialization would fail and set the 
+  did not explicitly request the mingw tool, mingw tool initialization would fail and set the
   default compiler to MSVC which wasn't installed, yielding broken build.
   Updated mingw tool so that the generate and exists methods use the same mingw search paths
   (issue #4134).
+- Fix issue where Express versions of the MSVC compiler were not detected due to differences
```

Comment:
Should this blurb also be in CHANGES.txt?

Reply:
Reworked the CHANGES.txt blurb to explicitly encompass what was fixed.